benchmarks/dynamo/common.py

```diff
@@ -737,11 +738,16 @@ def maybe_mark_profile(*args, **kwargs):
         for k, v in kwargs["dynamo_stats"].items():
             headers.append(k)
             row.append(v)
-    output_csv(
-        output_filename,
-        headers,
-        row,
-    )
+    if (
+        torch.distributed.is_available()
+        and torch.distributed.is_initialized()
+        and torch.distributed.get_rank() == 0
+    ):
+        output_csv(
```

Comment:
we still need to output_csv for single gpu, this gonna break CI dashboard

Reply:
oh, right, good catch, let me change the check.